http/client.go

```diff
@@ -110,6 +110,9 @@ func (c *RestClient) Request(method, path string, body io.Reader, header http.He
 	switch resp.Header.Get("Content-Encoding") {
 	case "gzip":
 		resp.Body, err = gzip.NewReader(resp.Body)
+		resp.Header.Del("Content-Encoding")
```

Comment:
Why removing something that was present in the response and used ? it seems to me that it will not affect the response at this point of the call path.

Reply:
ok I found a cleaner why to achieve the same behavior as explain here (https://golang.org/pkg/net/http/):

```
/ Uncompressed reports whether the response was sent compressed but
        // was decompressed by the http package. When true, reading from
        // Body yields the uncompressed content instead of the compressed
        // content actually set from the server, ContentLength is set to -1,
        // and the "Content-Length" and "Content-Encoding" fields are deleted
        // from the responseHeader. To get the original response from
        // the server, set Transport.DisableCompression to true.
        Uncompressed bool
```

I set `resp.Uncompressed = true` now